packages/core/strapi/src/cli/commands/transfer/command.ts

```diff
@@ -44,14 +44,128 @@ const command = () => {
+            // Log environment variables status
+            const hasUrl = Boolean(process.env.STRAPI_TRANSFER_URL);
+            const hasToken = Boolean(process.env.STRAPI_TRANSFER_TOKEN);
+
+            console.info('ℹ️  Data transfer documentation: https://docs.strapi.io/dev-docs/data-management/transfer');
+
+            if (hasUrl && hasToken) {
+              console.info('ℹ️  Found transfer configuration in your environment:');
+              console.info('   → STRAPI_TRANSFER_URL environment variable value will be used as default URL');
+              console.info('   → STRAPI_TRANSFER_TOKEN environment variable value will be used as default token');
+            } else if (hasUrl || hasToken) {
+              if (hasUrl) {
+                console.info('ℹ️  Found partial transfer configuration:');
+                console.info('   → STRAPI_TRANSFER_URL environment variable value will be used as default URL');
+                console.info('   → STRAPI_TRANSFER_TOKEN environment variable not found ');
+              } else {
+                console.info('ℹ️  Found partial transfer configuration:');
+                console.info('   → STRAPI_TRANSFER_URL environment variable not found');
+                console.info('   → STRAPI_TRANSFER_TOKEN environment variable value will be used as default token');
+              }
+            } else {
+              console.info('ℹ️  No transfer configuration found in environment variables');
+              console.info('   → Add STRAPI_TRANSFER_URL and STRAPI_TRANSFER_TOKEN environment variables to make the transfer process faster for future runs');
+            }
+            console.log(); // Empty line for better readability
+
+            const { direction } = await inquirer.prompt([
+              {
+                type: 'list',
+                name: 'direction',
+                message: 'Choose transfer direction:',
+                choices: [
+                  { name: 'Pull data from remote Strapi to local', value: 'from' },
+                  { name: 'Push local data to remote Strapi', value: 'to' }
+                ]
+              }
+            ]);
+
+            if (direction === 'from') {
+              const { remoteUrl, token } = await inquirer.prompt([
+                {
+                  type: 'input',
+                  name: 'remoteUrl',
+                  message: 'Enter the URL of the remote Strapi instance to get data from:',
+                  default: process.env.STRAPI_TRANSFER_URL,
+                  validate(input) {
+                    try {
+                      const url = new URL(input);
+                      if (!['http:', 'https:'].includes(url.protocol)) {
+                        return 'URL must use http: or https: protocol';
+                      }
+                      return true;
+                    } catch (error) {
+                      return 'Please enter a valid URL (e.g., http://localhost:1337 or https://example.com)';
```

Comment:
```suggestion
                      return 'Please enter a valid URL (e.g., http://localhost:1337/admin or https://example.com/admin)';
```
And eventually we need to make it retry with /admin if it doesn't have it and the initial connection fails